app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -174,6 +177,7 @@ export class EcsServerGroupConfigurationService {
     command.backingData = {
       // terminationPolicies: clone(this.terminationPolicies)
       launchTypes: clone(this.launchTypes),
+      networkModes: clone(this.networkModes),
```

Comment:
is adding this here actually a new thing, or was it always here and we just didn't have to define it before b/c the JS implementation didn't need a type explicitly defined?

Reply:
In the JS, it was directly being referred to. Rather than having it as backing data, it was just directly referring to a static list in the JS.